homeassistant/components/switchbot/coordinator.py

```diff
@@ -74,12 +77,12 @@ def _async_handle_bluetooth_event(
                 self._ready_event.set()
             _LOGGER.debug("%s: Switchbot data: %s", self.ble_device.address, self.data)
             self.device.update_from_advertisement(adv)
-        self.async_update_listeners()
+        super()._async_handle_bluetooth_event(service_info, change)
 
     async def async_wait_ready(self) -> bool:
         """Wait for the device to be ready."""
         with contextlib.suppress(asyncio.TimeoutError):
-            async with async_timeout.timeout(55):
+            async with async_timeout.timeout(DEVICE_STARTUP_TIMEOUT):
```

Comment:
Is it intentional to change the timeout from 55 to 30s?

Reply:
Yes.  It was way too high.  It was left over from when we had a much less reliable bluetooth stack